### PR TITLE
Feature 4 1 modalsignup

### PR DESCRIPTION
**Pull request by almedaria**
Updated signup.css and added close button

### CODE REVIEW

**Comment by kbcacas** [comment]
Looks good, very user-friendly

File: src/components/SignUp/SignUp.css [hunk 1]
```diff
@@ -24,6 +24,7 @@
 .modal-img {
   width: 100%;
   height: 100%;
+  object-fit: cover;
```
Comment (ccases) [inline src/components/SignUp/SignUp.css:27]: this changes the warped image. nice

File: src/components/SignUp/SignUp.css [hunk 2]
```diff
@@ -48,7 +48,7 @@ button {
   border: none;
 }
 
-.close-button {
+.close-modal-button {
```
Comment (ccases) [inline src/components/SignUp/SignUp.css:51]: improves UX, nice!

**Review by ccases — APPROVED** [review]

File: src/components/SignUp/SignUp.js [hunk 3]
```diff
@@ -125,6 +126,11 @@ function SignUp(props) {
                     />
                   </form>
                 </div>
+                <MdClose
```
Comment (kbcacas) [inline src/components/SignUp/SignUp.js:129]: great use of MdClose

**Review by kbcacas — APPROVED** [review]

**Pull request merged**